main.js

```diff
@@ -14,10 +14,18 @@ const params = new URLSearchParams(document.location.search);
 const url = params.get('url');
 const east = parseFloat(params.get('east'));
 const north = parseFloat(params.get('north'));
+const img_type = params.get('type');
+
+const img_type_tag = document.getElementById('image_type');
+(img_type === 'ortho') ? img_type_tag.innerText = "Type d'image: Orthophoto" : img_type_tag.innerText = "Type d'image: Image aérienne";
 
 const img = url.split('/').slice(-1);
 const h5 = document.getElementById('image_id');
-h5.innerText = "Image: " + img;
```

Comment:
Using template strings in JS is more readable. You can prepare a <div> with an id inside your h5 and then in one line
 myDiv.innertHTML = `Image: ${img} <a target="_blank" href="${url}">${img}</a>`